ssapy/io.py

```diff
@@ -1159,23 +1234,26 @@ def save_csv_line(file_name, df, sep='\t', dtypes=None):
     return
 
 
-# Create a lock to synchronize access to the file
-import threading
-file_lock = threading.Lock()
+_column_data = None
+def exists_in_csv(csv_file, column, number, sep='\t'):
```

Comment:
Add docstring

Reply:
This function is not thread safe as opposed to what was there before. Can you add back in the thread safety features?

Reply:
I had added that file_lock but I did not understand what I was exactly doing, if that is the correct way to create a lock I can reimplement it.

Reply:
I didn't check that. If you're the one that added the lock and your simply removing that feature, then okay. If you want thread safety, that's a longer conversation.